### PR TITLE
added workplan based off kick-off meeting notes

### LINKED ISSUE #16: Create WORKPLAN.md document
Create a WORKPLAN.md document with the outputs and remaining questions for the 1 year work scope. See the meeting notes from the kick off meeting.

### PR DESCRIPTION
**Pull request by cmharlow**
Will close #16

### CODE REVIEW

File: WORKPLAN.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## Evolving Work Plan
+
+1. [Gather/Identify Use Cases (2 months, June - July):](https://github.com/LD4P/ld4-community-recon/wiki/Entity-Resolution,-Reconciliation,-and-Lexicalization-Use-Cases-Work)
+    - Identify common use cases for reconciliation/resolution as a group;
+    - Make use cases expressed in a common way (i.e. shared template);
+    - Work on an accelerated time line for recon use cases;
+    - Pull in idea of context & standards as applicable.
+3. [Assessment & discussion of gathered use cases to split up among work areas (2-3 months, August - October):](https://github.com/LD4P/ld4-community-recon/wiki/Entity-Resolution,-Reconciliation,-and-Lexicalization-Use-Cases-Work)
+    - 1. algorithms and approaches to matching;
+    - 2. abstract out workflow & identify shared pieces;
```
Comment (ajs6f) [inline WORKPLAN.md:17]: `workflow` => `workflows`?

File: WORKPLAN.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - 3. Other work areas as identified;
+4. Develop or identify (multi-tracks?) tools for the work areas surfaced (TBD);
+   - Steps TBD based on upstream work;
+   - Tools to be implement across institutions;
```
Comment (ajs6f) [inline WORKPLAN.md:21]: typo `implemented`, or possibly `Tools to implement across institutions` (active voice FTW).

File: WORKPLAN.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Evolving Work Plan
+
+1. [Gather/Identify Use Cases (2 months, June - July):](https://github.com/LD4P/ld4-community-recon/wiki/Entity-Resolution,-Reconciliation,-and-Lexicalization-Use-Cases-Work)
+    - Identify common use cases for reconciliation/resolution as a group;
+    - Make use cases expressed in a common way (i.e. shared template);
+    - Work on an accelerated time line for recon use cases;
+    - Pull in idea of context & standards as applicable.
+3. [Assessment & discussion of gathered use cases to split up among work areas (2-3 months, August - October):](https://github.com/LD4P/ld4-community-recon/wiki/Entity-Resolution,-Reconciliation,-and-Lexicalization-Use-Cases-Work)
+    - 1. algorithms and approaches to matching;
```
Comment (ajs6f) [inline WORKPLAN.md:16]: add `matched to use cases`?

**Review by ajs6f — APPROVED** [review]
Nice!

**Comment by cmharlow** [comment]
going to revert/re-open so we can get the requested changes in and give others a chance to review, comment, etc. (especially in view of recent questions about work scope vs definitions in #30 )

**Pull request merged**